cartography/data/jobs/scoped_analysis/aws_ec2_iaminstanceprofile.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "EC2 Instances assume IAM roles",
```

Comment:
previously we assumed that ec2instance-->iamrole if the names matched. this is very incorrect, so we now implement iaminstanceprofiles correctly, and we also keep the same interface of still letting the user query for ec2 instances that can assume roles